lab-edward/lib/cowsayCli.js

```diff
@@ -0,0 +1,4 @@
+var cowsay = require('../lib/cowsay');
+var greeting = process.argv.slice(2).join(' ');
```

Comment:
Nice! At first I was wondering why you were slicing and joining, but now I see that you wanted flexibility in including multiple arguments.